Source/NSScroller.m

```diff
@@ -130,6 +133,12 @@ + (CGFloat) scrollerWidthForControlSize: (NSControlSize)controlSize
   return scrollerWidth;
 }
 
++ (NSScrollerStyle)preferredScrollerStyle
+{
+  // FIXME: a theme should define this?
```

Comment:
No need to change the comment but: not necessarily the theme.

Either way we do not support any other style of scrollers at this time, so you can optionally remove the comment, too :)